devoxx-cfp-stepengine/src/main/java/org/tweetwallfx/devoxx/cfp/stepengine/dataprovider/ScheduleDataProvider.java

```diff
@@ -94,7 +94,7 @@ public DataProvider create(final StepEngineSettings.DataProviderSetting dataProv
     }
 
     /**
-     * POJO used to configure {@link TopTalksWeekDataProvider}.
+     * POJO used to configure {@link ScheduleDataProvider}.
```

Comment:
Shouldn't the change require an update of the years in the header? Can be done when releasing though

Reply:
Done